### DISCUSSION TITLE
How to fix: Loading .env. This will not be loaded when running the server in production.

### DISCUSSION THREAD

**Discussion by HendrikJan** [nuxt/bridge#517] [Q&A] [answered]
When I start my server using `"nuxi start"` I get this message: `Loading .env. This will not be loaded when running the server in production.`

And indeed, when starting the server using `"nodejs ./.output/server/index.mjs"` the environment is not being loaded.

What is the recommended way to load .env on the server?

**Comment by danielroe** [comment, marked as answer]
Depends on the kind of server (e.g. Linux, Windows, serverless, etc.). But you can run  `source .env && node .output/server/index.mjs` on a POSIX shell (Mac/Linux).

**Reply by HendrikJan** [reply]
This does work indeed on Linux.
I had to remove spaces around "=" signs for this to work.
